### PR TITLE
Add slot numbers in lobby

### LINKED ISSUE #1268: Number player slots
Players can be move "to slot #", however we always have to count the slots.

Display player slot numbers next to the slot.

### PR DESCRIPTION
**Pull request by Sheeo**
This facilities easier moving of players while in the lobby, using the
dropdown menu.

In the future we'd love to have drag&drop, but that's somewhat more
involved.

Closes #1268

### CODE REVIEW

**Comment by aeoncleanse** [comment]
This doesn't work, it crashes the whole game... ...



Kidding :p It works perfectly, as expected! :+1:

File: lua/ui/lobby/lobby.lua [hunk 1]
```diff
@@ -2005,6 +2005,10 @@ function CreateSlotsUI(makeLabel)
     labelGroup.Height:Set(21)
     LayoutHelpers.AtLeftTopIn(labelGroup, GUI.playerPanel, 5, 5)
 
+    local slotLabel = makeLabel("", 14)
+    labelGroup:AddChild(slotLabel)
+    Tooltip.AddControlTooltip(slotLabel, 'rating')
```
Comment (Sheeo) [inline lua/ui/lobby/lobby.lua]: Obviously the tooltip here should be "Slot Number"

**Comment by aeoncleanse** [comment]
Fixed the column title position, added tooltip, and removed non-functional code in the release branch.

**Comment by aeoncleanse** [comment]
31c560bc0012393b7aa2feb717ffeb529f055fd9

**Pull request closed without merging**